tasks/cluster_agent.py

```diff
@@ -116,10 +121,16 @@ def integration_tests(ctx, install_deps=False, race=False, remote_docker=False,
 
 
 @task
-def image_build(ctx, arch='amd64', tag=AGENT_TAG, push=False):
+def image_build(ctx, arch=None, tag=AGENT_TAG, push=False):
     """
     Build the docker image
     """
+    if arch is None:
+        arch = CONTAINER_PLATFORM_MAPPING.get(platform.machine().lower())
+
+    if arch is None:
+        print("Unable to determine architecture to build, please set `arch` parameter")
+        raise Exit(code=1)
```

Comment:
should we set `amd64` in that case to have the previous behavior ?

Reply:
I trust Python `platform` to return the right value. If it does not there's probably something fishy somewhere.